core/test/results/sample_v2.json

```diff
@@ -2502,6 +2502,9 @@
       "score": null,
       "scoreDisplayMode": "informative",
       "displayValue": "2 long tasks found",
+      "metricSavings": {
+        "TBT": 1220.691999999999
```

Comment:
Drive-by fix, this should have been updated in #15197 but our passing tests were stale